models.py

```diff
@@ -289,3 +296,14 @@ class UtilityModelView(RestrictedModelView):
     form_args = dict(
         bank_description = dict(validators=[bank_description_check])
     )
+
+    column_formatters = {
+        'fields_description':  _formatter
+    }
+
+    def on_model_change(self, form, model, is_created):
+        opts = jsbeautifier.default_options()
+        opts.indent_size = 2
+        if is_created:
```

Comment:
these two lines are not required

Reply:
you meant the opts?

Reply:
you removed the wrong lines

```
if is_created:
   model.bank_description = jsbeautifier.beautify(form.bank_description.data)
```

these lines are not useful

Reply:
Re-added then deleted the if lines